zzzz_modular_occulus/code/game/gamemodes/occultist/occultist_weapons.dm

```diff
@@ -1,17 +1,173 @@
 /obj/item/weapon/tool/sword/cult
 	name = "Non-Euclidean Blade"
+	spawn_blacklisted = TRUE
+
+	desc = "Standing around, staring at this for what are you? To killing get!"
+	icon = 'zzzz_modular_occulus/icons/obj/occultist_weap/occsword.dmi'
+
+	icon_state = "non-euclidian_blade"
+	//item_state = "righthand"
+
+	matter = list(MATERIAL_STEEL = 1, MATERIAL_OH_GOD_HELP_ME_I_AM_TRAPPED_IN_HERE = 15, MATERIAL_LEATHER = 5) //This is messed up on purpose. Don't fix it. It's supposed to cause wierd shit
+	sharp = FALSE
+	edge = FALSE
+	w_class = ITEM_SIZE_NORMAL
+	slot_flags = SLOT_BELT | SLOT_BACK
+	worksound = WORKSOUND_HARD_SLASH
+	force = WEAPON_FORCE_PAINFUL
+	throwforce = WEAPON_FORCE_PAINFUL
+	tool_qualities = list(QUALITY_PULSING = 7, QUALITY_SCREW_DRIVING = 3)
+	item_icons = list(
+		slot_l_hand_str = 'zzzz_modular_occulus/icons/obj/occultist_weap/occsword.dmi',
+		slot_r_hand_str = 'zzzz_modular_occulus/icons/obj/occultist_weap/occsword.dmi',
+		slot_belt_str = 'zzzz_modular_occulus/icons/obj/occultist_weap/occsword.dmi',
+		slot_back_str = 'zzzz_modular_occulus/icons/obj/occultist_weap/occsword.dmi')
+	item_state_slots = list(
+		slot_l_hand_str = "lefthand",
+		slot_r_hand_str = "righthand",
+		slot_belt_str = "belt",
+		slot_back_str = "back"
+		)
+
+/obj/item/weapon/tool/sword/cult/attack(mob/living/carbon/human/M, mob/user)
+	if(istype(M, /mob/living/carbon/human))
+		if(M.sanity)
+			M.sanity.onPsyDamage(36)
+			to_chat(M, SPAN_DANGER(M.sanity.pick_quote_20()))
+			M.adjust_hallucination(10, 10)
+	..()
+	return
+
+
+/obj/item/weapon/tool/hammer/homewrecker/cult
+	name = "Soulcrusher"
+
+	icon = 'zzzz_modular_occulus/icons/obj/occultist_weap/occhammer.dmi'
+
+	desc = "Heavy extremely sledgehammer. A Handle resembles a chunk welded to which a large long steel."
```

Comment:
Heavy extremely sledgehammer indeed, however you aren't Peter Gabriel. Is this a typo?

Reply:
No. The descriptions are intentionally fucked up.